File: flight_system_front/src/models/booking.ts
export class Booking {
    booking_id: string;
    flight_id: string;
    passenger_name: string;
    seat_number: string;

    constructor(booking_id: string, flight_id: string, passenger_name: string, seat_number: string) {
        this.booking_id = booking_id;
        this.flight_id = flight_id;
        this.passenger_name = passenger_name;
        this.seat_number = seat_number;
    }
}

File: entities/Booking.py
from pydantic import BaseModel
from uuid import UUID


class Booking(BaseModel):
    flight_id: UUID
    booking_id: UUID
    passenger_name: str
    seat_number: str

    def to_json(self) -> dict:
        return self.model_dump_json()
    
    @staticmethod
    def to_json(flight_id, booking_id, passenger_name, seat_number) -> dict:
        return {
            "flight_id": flight_id,
            "booking_id": booking_id,
            "passenger_name": passenger_name,
            "seat_number": seat_number
        }